book/src/requirements/apple.md

```diff
@@ -0,0 +1,64 @@
+# macOS & iOS Requirements
+
+Unless the "fips" feature is enabled, building aws-lc-rs v1.7.0 (or later) for the following targets should only require
```

Comment:
```suggestion
Unless building for ios or the "fips" feature is enabled, building aws-lc-rs v1.7.0 (or later) for the following targets should only require
```

Also, is FIPS buildable for `aarch64-apple-ios`? I think AWS-LC supports it since we had to do some ACVP testing for ios.

Reply:
I think so, but this sentence is saying that only a C/C++ compiler is required, which is not true for iOS.

Reply:
You're right. Updated in the latest commit.